DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/ViewDelegateControllerTests.cs

```diff
@@ -67,5 +67,41 @@ public void Deactivating_delegate_on_wrong_centre_returns_not_found_result()
             // then
             result.Should().BeNotFoundResult();
         }
+
+
+        [Test]
+        public void Reactivating_delegate_returns_redirect()
+        {
+            // given

```

Comment:
Could we change the comments in these tests to be capitalised, so they are _Given_, _When_, _Then_ ?